content/blog/2024/frx/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "`frx-challenges`: A new tool to host data challenges for Frictionless Research Exchanges"
+date: 2024-12-06
+tags: [open source]
+categories: [impact]
+---
+
+2i2c is pleased to announce the `frx-challenges` project, a new open source tool to help communities host data challenges on shared infrastructure:
+
+https://github.com/2i2c-org/frx-challenges
+
+This project aims to make it easier for administrators to provide a service that enables users to **submit code and data** that are **evaluated on secure infrastructure with access to private data and resources**. It also provides a leaderboard that helps users compare their performance against others.
+
+{{< figure
+  src="images/leaderboard.png"
+  width="75%"
+  caption="An example leaderboard for a data challenge, taken from the [Cellmap Challenge](https://cellmapchallenge.janelia.org/). Users make submissions that are run against secure and private infrastructure and data, and provides feedback about the submission's performance. Learn more about the FRX challenges project here: https://2i2c.org/frx-challenges/"
+>}}
+
+It is designed to be lightweight and flexible, and can be run on a variety of shared infrastructure. For those who wish to run this project on cloud infrastructure, we've also published a Helm Chart to help you deploy `frx-challenges` with Kubernetes:
+
+https://2i2c.org/frx-challenges-helm-chart/
```

Comment:
I would suggest not putting this so prominently, as it doesn't actually have any useful info right now on how to use it.